src/app/src/pages/Admin.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from "react"
+import { Box, List, ButtonGroup, Button , Badge, Typography, Divider} from "@mui/material"
+import NavBar from "../components/NavBar"
+import AdminRequestCard from "../components/AdminRequestCard"
+import AdminAuthorCard from "../components/AdminAuthorCard"
+import AdminPostCard from "../components/AdminPostCard"
+import AdminNodeCard from "../components/AdminNodeCard"
+
+export default function Admin(): JSX.Element {
+    const [listDisplay, setListDisplay] = React.useState({title:'Requests',id:0});
+
+    //Some fake data to help with layouts
+    const signupRequests = [
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        displayName:"Lara Croft",
+        },
+        {
+        id:"c3293ed4-e55e-4986-8311-5ad43a27f5a3",
+        displayName:"Nathan Drake",
+        },
+    ];
+
+    const authors = [
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        displayName:"Lara Croft",
+        profileImage:null,
+        },
+        {
+        id:"c3293ed4-e55e-4986-8311-5ad43a27f5a3",
+        displayName:"Nathan Drake",
+        profileImage:null,
+        },
+    ];
+
+    const posts=[
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        author:authors[0],
+        date: "2022-02-25"
+        }
+    ];
+
+    const nodes=[
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        username:"NodeOne",
+        }
+    ];
+
+    // Get length for badges
+    const totalRequests = signupRequests.length;
+    const totalAuthors = authors.length;
+    const totalPosts = posts.length;
+    const totalNodes = nodes.length;
+
+    // Sidebar Button group
+    const buttons = [
```

Comment:
Can generalize the "sx" prop by making an object!

Reply:
Okay! I applied some updates based on this comment and the one for the fffr page.